cmake/apple/install_apple.sh.in

```diff
@@ -21,20 +21,16 @@ MSG_COLOR='\x1B[1;36m'
 COLOR_RESET='\x1B[0m'
 echo -e "$MSG_COLOR\n\nCreating App Bundle \"$APP\"...$COLOR_RESET"
 
-# Locate macdeployqt, assume homebrew & Qt5
-which macdeployqt > /dev/null 2>&1
```

Comment:
> I don't have a clear idea of what MacPorts is.

It's an alternative to homebrew.

`macdeployqt` was already on the `$PATH` in MacPorts, so that was the way of supporting both package managers.